src/sentry/runner/commands/backup.py

```diff
@@ -54,6 +54,9 @@ class ComparatorFindings:
     def __init__(self, findings: list[ComparatorFinding]):
         self.findings = findings
 
+    def concat(self, findings: list[ComparatorFinding]) -> None:
+        self.findings += findings
```

Comment:
is extend better here?
```suggestion
        self.findings.extend(findings)
```